ui/app/src/modules/Preview/Tools/ContentTree.tsx

```diff
@@ -349,18 +352,17 @@ function TreeItemCustom(props: TreeItemCustomInterface) {
       }}
     >
       {
-        nodes.children.map((node) => (
-          <TreeItemCustom key={node.id} nodeId={node.id} {...props} />
-        ))
+        nodes.children?.map(_nodeId => <TreeItemCustom {...props} key={String(_nodeId)} nodeId={String(_nodeId)}/>)
```

Comment:
use childNodeId as argument name
The need to cast is likely to come from improper typing